edx2bigquery/bqutil.py

```diff
@@ -848,6 +848,8 @@ def get_job_config(schema, source_format):
 
     job_config.schema = list(get_configuration_schema(schema))
     job_config.write_disposition = BIGQUERY_WRITE_DISPOSITION
+    # 50 is an arbitrary number.
+    job_config.max_bad_records = 50
```

Comment:
Let's define a module constant like `BIGQUERY_MAX_BAD_RECORDS=50` and then:
```suggestion
    job_config.max_bad_records = BIGQUERY_MAX_BAD_RECORDS
```

Reply:
Addressed @diegomillan